<?xml version="1.0" encoding="UTF-8"?>
<testng-results failed="1" passed="1" skipped="0" total="2">
<suite name="Cucumber" duration-ms="17">
<test name="Cucumber" duration-ms="17">
<class name="Global hooks">
<test-method name="A passing scenario" status="PASS" duration-ms="3" started-at="1970-01-01T00:00:00.005Z" finished-at="1970-01-01T00:00:00.008Z"/>
<test-method name="A failing scenario" status="FAIL" duration-ms="3" started-at="1970-01-01T00:00:00.009Z" finished-at="1970-01-01T00:00:00.012Z">
<exception class="Error">
<message><![CDATA[
When a step fails...........................................................failed
]]></message>
<full-stacktrace>
<![CDATA[Error: Exception in step
samples/global-hooks/global-hooks.feature:10]]>
</full-stacktrace>
</exception>
</test-method>
</class>
</test>
</suite>
</testng-results>
